packages/config/config/devices/0x001d/rzi10-1l.json

```diff
@@ -0,0 +1,26 @@
+// Leviton RZI10-1L
+// Dimmer switch 1000w incandescent
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "RZI10-1L",
+	"description": "Dimmer switch 1000w incandescent",
+	"devices": [
+		{
+			"productType": "0x0501",
+			"productId": "0x0206"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"description": "Association group 1",
```

Comment:
Change "Association group 1" to "Lifeline"

Reply:
Better remove it - it adds no value over the label

Reply:
Remove this line.